index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles-conference/stylesheets/styles.css">
+  <title>Swing Dance Styles and Steps Library</title>
+</head>
+<body>
+  <header>
+    <div class="popup-menu invisible">
+      <img class="close-button" src="styles-conference/assets/Icon-Cancel.svg" alt="">
+      <ul class="popup-menu-list">
+        <li><a href="./about.html">About</a></li>
+        <li><a href="./index.html#program">Program</a></li>
+        <li><a href="#AJ">Join</a></li>
+        <li><a href="#sponsors">Sponsor</a></li>
+        <li><a href="#news">News</a></li>
+        <li><button class="sdssl-button">SDSSL Campaign</button></li>
+      </ul>
+    </div>
+    <nav>
+      <div class="dark-header">
+        <div class="media">
+          <a href="https://www.facebook.com/martinezequiel.gonzalez.5"><img src="styles-conference/assets/fb_icon.png" alt="facebook_icon"></a>
+          <a href="https://twitter.com/MarezegonZ"><img src="styles-conference/assets/tw_icon.png" alt="twitter_icon"></a>
+        </div>
+        <div class="dark-menu-list">
+          <a href="#">Logout</a>
+          <a href="#">Mypage</a>
+          <a href="#">Spanish</a>
+        </div>
+      </div>
+      <div class="navbar toggle-selection">
+        <a class="ham-link"href="#">
+          <img class= "ham-img" src="styles-conference/assets/ham.png" alt="">
+        </a>
+        <a href="#"><img class="logo-navbar" src="styles-conference/assets/sd_logo.png" alt="logo_sd"></a>
+        <ul class="menu-list-desktop">
+          <li><a class="home-about" href="./about.html">About</a></li>
+          <li><a class="menu-item-desktop" href="#program">Program</a></li>
+          <li><a class="menu-item-desktop" href="#AJ">Join</a></li>
+          <li><a class="menu-item-desktop" href="#sponsors">Sponsor</a></li>
+          <li><a class="menu-item-desktop" href="#news">News</a></li>
+          <li><button class="sdssl-button">SDSSL Campaign</button></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <section class="home toggle-selection">
+    <section class="welcome-page">
+      <div class="headings">
+        <h2 class="hello">"Hello! Swing Dancers"</h2>
+        <h1 class="title">SWING DANCE STYLES AND STEPS LIBRARY 2023</h1>

```

Comment:
- [ OPTIONAL ] I highly recommend also here to update it and to have an h1 before h2 to follow the best practice in HTML rules so please fix it and I am assigning as optinal because the privous reviewer didn't mention it here 🚀💯🤩

Reply:
Done! Thanks @Reem-lab